src/network/network_survey.cpp

```diff
@@ -329,6 +355,7 @@ std::string NetworkSurveyHandler::CreatePayload(Reason reason, bool for_preview)
 		auto &info = survey["info"];
 		SurveyOS(info["os"]);
 		info["os"]["hardware_concurrency"] = std::thread::hardware_concurrency();
+		info["os"]["memory"] = MemoryToText(info["os"]["memory"]);
```

Comment:
I guess there's some magic going on here by the json stuff, to be able to pass a 64bit value without casts and return a std::string into the same place.

Reply:
Absolutely! If you like, I can avoid it, but this is what nlohmann-json allows just fine :)

Reply:
It annoyed me enough that I fixed it. Much more proper now.